Library/Homebrew/extend/ENV/super.rb

```diff
@@ -67,6 +67,11 @@ def setup_build_environment(formula = nil)
     self["HOMEBREW_INCLUDE_PATHS"] = determine_include_paths
     self["HOMEBREW_LIBRARY_PATHS"] = determine_library_paths
 
+    if MacOS::Xcode.without_clt?
+      self["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version.to_s
+      self["SDKROOT"] = MacOS.sdk_path
```

Comment:
I changed it after I noticed that setting a `/` `SDKROOT` caused cmake to emit warnings. It isn't necessary for builds to work.

`.without_clt?` returns true only for Xcode installed, greater than 4.3, with no CLT. If `Xcode.installed?` is true there should be a usable SDK.